### PR TITLE
Add display_set to answer serializer

### PR DESCRIPTION
**Pull request by chrisvanrun**
Small addition for working with display sets in CIRRUS

### CODE REVIEW

File: app/grandchallenge/reader_studies/serializers.py [hunk 1]
```diff
@@ -143,6 +143,10 @@ class AnswerSerializer(HyperlinkedModelSerializer):
     answer_image = HyperlinkedRelatedField(
         read_only=True, view_name="api:image-detail"
     )
+    display_set = HyperlinkedRelatedField(
+        queryset=DisplaySet.objects.all(),
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/serializers.py:147]: We need to check that the user has permission to create answers for this `display_set` if this field is not read only. This `queryset` (and the one for the images and questions) need to be filtered.

Comment (chrisvanrun) [reply app/grandchallenge/reader_studies/serializers.py:147]: Definitely needs to be read_only! I'll add some tests to also cover the other members.

Although I see that those tests are rendering entire views. So those are ready to be optimized first.

Comment (jmsmkn) [reply app/grandchallenge/reader_studies/serializers.py:147]: This is quite tricky, so maybe @MikeOverkamp-diag can help.

**Comment by jmsmkn** [comment]
I'm going to close this as I think it needs a lot of work, please feel free to re-open when ready.

**Pull request closed without merging**